### PR TITLE
refactor: adjust delegates list page

### PR DESCRIPTION
**Pull request by shahin-hq**
## Summary
Closes: https://app.clickup.com/t/86du08kag

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by shahin-hq** [comment]
> Is the "Validated name" correct here or an oversight from the designs? Shouldn't it be delegate, considering that we are in develop branch? 2024-09-03-123839_402x156_scrot

https://www.figma.com/design/o6b1tEMOdUtHv7ijVs6pMX/ARKVault?node-id=13929-58239&node-type=TEXT&m=dev

it is `validated name` here too, will ask Oleg to clarify

image

**Comment by ItsANameToo** [comment]
> > Is the "Validated name" correct here or an oversight from the designs? Shouldn't it be delegate, considering that we are 

"validated" is wrong either way, it should be `Validator Name`. In this case since it targets develop it will need to be `Delegate Name`. General rule: when it is about delegates/validators mind the branch you work on to determine the correct word, and make it configurable to handle the Mainsail network (similar to what is done in other areas where it swaps between delegate/validator depending on network).

**Review by samharperpittam — CHANGES_REQUESTED** [review]
Looking good. Only minor issue I can find is that the padding shown in the image below should be 24px

image


Design
image

File: src/domains/vote/components/AddressTable/AddressRow/AddressRowMobile.test.tsx [hunk 1]
```diff
@@ -351,6 +352,28 @@ describe("AddressRowMobile", () => {
...
+			{
+				history,
+				route: `/profiles/${profile.id()}/votes`,
+			},
+		);
+
+		const historySpy = vi.spyOn(history, "push");
+
+		// eslint-disable-next-line testing-library/no-container
+		const containerDiv = container.querySelector("tr td > div") as HTMLDivElement;
```
[the remark this answers is not in the document]
Comment (shahin-hq) [reply src/domains/vote/components/AddressTable/AddressRow/AddressRowMobile.test.tsx]: Thanks. Fixed.

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**